### DISCUSSION TITLE
Error updating WindowsAppSdk up to 1.2.221209.1

### DISCUSSION THREAD

**Discussion by mcNets** [unoplatform/uno#11042] [Q&A] [answered]
I build a new project (Mobile + Skia + Wasm + Windows) and then I updated the packages of the windows head.

After installing WindowsAppSdk 1.2.221209.1 I received the next errors:

>"...\.nuget\packages\microsoft.windowsappsdk\1.2.221209.1\buildTransitive\Microsoft.InteractiveExperiences.Common.targets" was not imported by "...\.nuget\packages\microsoft.windowsappsdk\1.2.221209.1\buildTransitive\Microsoft.InteractiveExperiences.targets" at (7,3), due to the file being invalid.

Pointing to: Microsoft.InteractiveExperiences.Common.targets

>"...\.nuget\packages\microsoft.windowsappsdk\1.2.221209.1\buildTransitive\Microsoft.InteractiveExperiences.Common.targets" could not be loaded. '<', hexadecimal value 0x3C, is an invalid attribute character. Line 15, position 65.

And this one pointing to:
```
<?xml version="1.0" encoding="utf-8"?>

<Project xmlns="http://schemas.microsoft.com/developer/msbuild/2003">
  <Import Project="$(MSBuildThisFileDirectory)Microsoft.InteractiveExperiences.Common.targets"/>
  <!-- CsWinRT transitive build -->
  <ItemGroup>
     <CsWinRTInputs Include="$(MSBuildThisFileDirectory)..\lib\$(Ixp-UAPTargetVersion)\*.winmd" />
  </ItemGroup>
</Project>
```

I tried by aligning Microsoft.Windows.SDK.BuildTools to the last build:
```
<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" RuntimeFrameworkVersion="10.0.22621.28" />
<FrameworkReference Update="Microsoft.Windows.SDK.NET.Ref" TargetingPackVersion="10.0.22621.28" />
```

But it doesn't solve the problem.

I am able to build the project using the previous version: 1.1.5

Then I tried to update to the last 1.2.xxxxx version again and I'm getting an error here:

Microsoft.InteractiveExperiences.Common.targets

```
 <!-- Set the UAP metadata to reference based on the target version -->
  <PropertyGroup>
    <Ixp-UAPTargetVersion Condition="'$(TargetPlatformVersion)' < '10.0.18362.0'">uap10.0.17763</Ixp-UAPTargetVersion>
    <Ixp-UAPTargetVersion Condition="'$(Ixp-UAPTargetVersion)' == ''">uap10.0.18362</Ixp-UAPTargetVersion>
  </PropertyGroup>
```
And a squiggling line under the symbol: `<` saying that:

>Character '<', hexadecimal value 0x3x is ilegal in XML attribute values.

 I was trying to generate a WinUI app using VS Template Studio and then adding the project to an existing UNO project. But VSTS uses  WinUIEX that, in the last version, requieres WindowsAppSDK 1.2.xxx

[TestPackages.zip](https://github.com/unoplatform/uno/files/10492091/TestPackages.zip)

**Comment by jeromelaban** [comment]
This looks like a WinAppSDK issue, we won't be able to do much about it. You may want to report it in the WinAppSDK repo.

**Comment by mcNets** [comment, marked as answer]
It can be solved by changing the TargetFramework in this way:

`<TargetFramework>net7.0</TargetFramework>`

`<TargetFramework>net7.0-windows10.0.22000.0</TargetFramework>`

Have a look at [this discord](https://discord.com/channels/372137812037730304/663434534087426129/1039946595438972928) chat.
